rtf

```diff
@@ -137,35 +137,40 @@ class Vagrant(Host):
   def reset_platform(name, hosts):
     """ Update or replace the Vagrantfile configuration for the given platform """
     lines = []
+    # parse Vagrantfile into the lines array (and update it)
     with open("Vagrantfile", "r+") as fd:
       updated = False
```

Comment:
updated is truly a flag and not a break indicator.
We continue processing input when updated is true, to retrieve remaining lines.